src/tinyformat.h

```diff
@@ -167,7 +167,7 @@ namespace tinyformat {
 class format_error: public std::runtime_error
 {
 public:
-    format_error(const std::string &what): std::runtime_error(what) {
+    format_error(const std::string &fmtwhat): std::runtime_error(fmtwhat) {
```

Comment:
My only concern is that we occasionally pull updates to tinyformat.cpp from upstream (eg #8274), so any trivial changes like this add to the maintainer's burden when merging. Not sure if @laanwj has an opinion.